membrane_curvature/base.py

```diff
@@ -0,0 +1,72 @@
+import numpy as np
+
+from membrane_curvature.lib.mods import derive_surface, mean_curvature, gaussian_curvature
+import numpy as np
+
+
```

Comment:
```suggestion
from MDAnalysis.analysis.base import AnalysisBase

class MembraneCurvature(AnalysisBase):
```
I think you're missing an actual class definition!